angular.json

```diff
@@ -76,19 +96,27 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "configurations": {
+            "da": {
+              "browserTarget": "ng-sofie:build:development,da"
+            },
+            "en": {
+              "browserTarget": "ng-sofie:build:development,en"
+            },
             "production": {
               "browserTarget": "ng-sofie:build:production"
             },
             "development": {
               "browserTarget": "ng-sofie:build:development"
             }
           },
-          "defaultConfiguration": "development"
+          "defaultConfiguration": "en"
```

Comment:
Do we need to have both danish and english for development purposes?
An alternative could be to keep the default configuration as development and then update production and development to have the browserTarget of the `en` configuration.

Reply:
I have made the suggested changes.